api.md

```diff
@@ -202,7 +202,8 @@ function.
 
 | Name              | Description                                   |
 |-------------------|-----------------------------------------------|
-| `if_not_attached` | Prevents error if data node is already attached to the hypertable. A notice will be printed that the data node is attached. |
+| `if_not_attached` | Prevents error if data node is already attached to the hypertable. A notice will be printed that the data node is attached. Defaults to `FALSE`. |
+| `repartition`     | Change the configuration so that all attached data nodes are used. Defaults to `TRUE`. |
```

Comment:
I haven't heard it is implemented. I guess you know better, since you were working on this.

Reply:
It was there when I looked at the code. Just documenting what is there.

Reply:
If it is not implemented, but only provided to the user, I think it is good to have a note in the documentation now. In a sense it is the part of documentation limitations.

Reply:
Not sure I follow. Looking at the code I see a reconfiguration of the hypertable in the code when `repartition` is true. It looks like this:

```
		if (repartition)
		{
			ts_dimension_set_number_of_slices(dim, num_nodes & 0xFFFF);

			ereport(NOTICE,
					(errmsg("the number of partitions in dimension \"%s\" was increased to %u",
							NameStr(dim->fd.column_name),
							num_nodes),
					 errdetail("To make use of all attached data nodes, a distributed "
							   "hypertable needs at least as many partitions in the first "
							   "closed (space) dimension as there are attached data nodes.")));
		}
```

Not sure what changes you want to have in the documentation.